pyexchange/korbit.py

```diff
@@ -361,4 +364,19 @@ def _format_pair_string(self, pair: str) -> str:
         if '-' in pair:
             return "_".join(pair.split('-')).lower()
         else:
-            return pair.lower()
\ No newline at end of file
+            return pair.lower()
+
+    def _choose_nonce(self) -> int:
```

Comment:
Please explain why threadsync is necessary.  Each method which requires a nonce is making a synchronous call, so concurrency is unexpected.

Reply:
This was added to resolve too many requests being fired off by the keeper at the same time, which have conflicting nonces from the perspective of the Korbit API, and then are subsequently rejected.

The approach is similar to that used in Kraken and Liquid: https://github.com/makerdao/pyexchange/commit/affec1616cc69edd08b63225b5fb9c1b88a6a423